File: Spiza/Gateways/Spiza.Gateways.Web/Models/Item.cs
    namespace Spiza.Gateways.Web.Models
{
    public class Item
    {
        public string Name { get; set; } = "";
        public string Category { get; set; } = "";
        public int Order { get; set; }
        public double Price { get; set; }
        public string Description { get; set; } = "";
        public string Image { get; set; } = "";
    }
}

File: Spiza/Frontends/Spiza.Mobile.Customer/spiza_customer/lib/models/item.dart
class Item {
  final String id;
  final String name;
  final String description;
  final String image;
  final String category;
  final num price;
  final int order;

  Item.fromJson(Map<String, dynamic> parsedJson)
      : id = parsedJson['id'],
        name = parsedJson['name'],
        description = parsedJson['description'],
        image = parsedJson['image'],
        price = parsedJson['price'],
        category = parsedJson['category'],
        order = parsedJson['order'];
}
